pandas/tests/io/test_parquet.py

```diff
@@ -71,6 +71,24 @@ def fp():
     return 'fastparquet'
 
 
+@pytest.fixture
+def fp_lt_014():
+    if not _HAVE_FASTPARQUET:
+        pytest.skip("fastparquet is not installed")
+    if LooseVersion(fastparquet.__version__) >= LooseVersion('0.1.4'):
+        pytest.skip("fastparquet is >= 0.1.4")
+    return 'fastparquet'
+
+
+@pytest.fixture
+def fp_ge_014():
```

Comment:
I don't think you are already using this one?

Reply:
correct. removed.